CMake/fletch-tarballs.cmake

```diff
@@ -121,24 +121,30 @@ if ( NOT fletch_ENABLE_OpenCV OR OpenCV_SELECT_VERSION VERSION_LESS 3.0.0 )
   unset(fletch_ENABLE_OpenCV_contrib CACHE)
 endif()
 
-if (OpenCV_SELECT_VERSION VERSION_EQUAL 3.1.0)
-  set(OpenCV_version "3.1.0")
+if (fletch_ENABLE_OpenCV OR fletch_ENABLE_ALL_PACKAGES)
+  set(OpenCV_version ${OpenCV_SELECT_VERSION})
```

Comment:
If it's a big rework, skip for now, it's ok to have the second block if that is the cleaner option.